lib/commands/workspace.js

```diff
@@ -4,12 +4,18 @@ const { contextMiddleware } = importLazy('../cli/context-middleware');
 const path = importLazy('path');
 const inquirer = importLazy('inquirer');
 const DevelopmentEnvironment = importLazy('../environment/development');
-const { platforms, uiModules, stripesModules, otherModules, allModules } = importLazy('../environment/inventory');
+const { allModules, allModulesAsFlatArray, fetchAndStoreModules } = importLazy('../environment/inventory');
 const { promptMiddleware } = importLazy('../cli/prompt-middleware');
 
 
 function workspaceCommand(argv) {
   const context = argv.context;
+
+  // When fetching, return immediately so that user can see the updated prompt on subsequent workspace invocations.
+  if (argv.fetch) {
```

Comment:
Given this returns so quickly, updating the inventory may be worthy of its own command.  Doing so may also reduce confusion above re: use of --modules and --fetch together.

Reply:
Sounds like a good way to move forward! I'll try and think of a good command name to use!